src/ZosJobsProvider.ts

```diff
@@ -35,12 +35,12 @@ export async function createJobsTree(log: Logger) {
     return tree;
 }
 
+// tslint:disable-next-line: max-classes-per-file
```

Comment:
We should try to avoid disabling tslint rules as much as possible.
Is this something we should discuss further, and maybe modify `tslint.json`?

Reply:
I do use the max-classes a lotso maybe I am doing something wrong by having so many